src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
The href attribute of the link tag should link to a CSS file, not a SCSS file. The browser does not understand SCSS syntax. You should compile your SCSS to CSS and then link to the generated CSS file.